test/device_firmware_update/check_dfu_i2c.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# The steps below are meant to be run on a Linux/macOS platform.
```

Comment:
How hard would it be to add specific instructions for building on Windows?  Is there more needed than simply replacing 'make' with 'ninja'?  For instance, does the command `cmake -B build --toolchain=xmos_cmake_toolchain/xs3a.cmake` change when building on Windows?

Reply:
I added the Windows instructions, the `cmake` command needed an update too.